src/Themes.js

```diff
@@ -1,5 +1,7 @@
 const Themes = {
   'Default' : {
+    appBackground: 'white',
+    appText: 'black',
```

Comment:
As default colors can be changed by user in browser settings or different, I prefer to set them